pythia/seo/tests/test_sunspotter.py

```diff
@@ -275,3 +290,21 @@ def test_hpc_to_hgs_position(sunspotter, obsdate):
 def test_get_lat_lon_in_hgs(sunspotter, obsdate):
     assert isinstance(sunspotter.get_lat_lon_in_hgs(obsdate)[0], Longitude)
     assert isinstance(sunspotter.get_lat_lon_in_hgs(obsdate)[1], Latitude)
+
+
+def test_match_with_swpc_for_obsdate_default(sunspotter, compare_df):
+    obsdate = '2005-12-31 12:48:02'
+    match_result = sunspotter.match_with_swpc_for_obsdate(obsdate)
+
+    for column in compare_df.columns:
+        assert all(np.isclose(match_result[column], compare_df[column]))
```

Comment:
I'm not too fun of having tests with asserts in a loop. If there's something wrong you will have to run it n times to get all the errors. Each time, run, find, fix, run, find, fix. Whereas running and getting all the errors at once is more handy.